editor/editor_settings.cpp

```diff
@@ -920,14 +922,32 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 void EditorSettings::_load_godot2_text_editor_theme() {
 	// Godot 2 is only a dark theme; it doesn't have a light theme counterpart.
 	_initial_set("text_editor/theme/highlighting/symbol_color", Color(0.73, 0.87, 1.0));
+	_initial_set("text_editor/theme/highlighting/symbol_bold", false);
+	_initial_set("text_editor/theme/highlighting/symbol_italic", false);
```

Comment:
This makes for a lot of boolean properties. I suggest merging them into a single enum with the following values:

- Regular
- Bold
- Italic
- Bold Italic